README.md

```diff
@@ -48,3 +48,63 @@ Requests to cross-scheme subresources will no longer send SameSite=Lax or SameSi
 
 For example, if http://site.example embeds an image from https://site.example, SameSite=Lax and SameSite=Strict cookies will no longer be sent on the request to https://site.example, and any such cookies in the response will be ignored. 
 
+## Questions
+### How do Schemeful Same-Site and Scheme-Bound Cookies differ?
```

Comment:
I feel like the first two sections of this are just summarizing what the proposals are, which is sort of redundant.

Reply:
Yeah, but the number of times we've had to answer this question anyway makes me think it might still be useful